package.json

```diff
@@ -103,7 +103,8 @@
   "jest": {
     "coverageDirectory": "<rootDir>/build/coverage",
     "collectCoverageFrom": [
-      "{components,helpers}/**/*.{ts,tsx,js}"
+      "{components,helpers}/**/*.{ts,tsx,js}",
+      "!helpers/{keycodes,testUtils}.{ts,tsx}"
```

Comment:
You might want to add a `sonar.coverage.exclusions` propriety to the `sonar-project.properties` files too. With those files referenced.